app/boomerang.config.js

```diff
@@ -5,6 +5,7 @@ angular.module('gdgXBoomerang')
 
     $routeProvider.
         when('/about', {templateUrl: 'app/about/about.html', controller: 'AboutController', controllerAs: 'vm'}).
+        when('/coc', {templateUrl: 'app/coc/coc.html', controller: 'COCController', controllerAs: 'vm'}).
```

Comment:
Let's call it `ConductController`. Same for files/dirs, let's do `/app/conduct/conduct.html`. I think it's clearer as to the meaning.